src/ui/ResultLayoutSelector/ValidLayout.ts

```diff
@@ -3,4 +3,4 @@
  *
  * See the [Result Layouts](https://developers.coveo.com/x/yQUvAg) documentation.
  */
-export type ValidLayout = 'list' | 'card' | 'table';
+export type ValidLayout = 'list' | 'card' | 'table' | 'preview';
```

Comment:
So I don't want to use any other layout since it gives a class to my result items with **a lot** of CSS applied to that style.
I also need to give a valid layout (line 164) to build from a template.

Is there a way around that?
Should I simply rename it to something like `other`?

I could move all my styling from the generated class (line 25 of the file `_QuerySuggestPreview.scss`) to a new class named `coveo-query-suggest-preview` and add it to the element.
This way even if a client use the `other` option, it will only render the template he gave and nothing else.